src/components/pages/HomePage.js

```diff
@@ -0,0 +1,39 @@
+import { React, useContext, useState, useEffect } from "react";
+import Navbar from "../NavBar";
+import Sidebar from "../SideBar";
+import { UserContext } from "../../constants/UserContext";
+
+const HomePage = () => {
+  const { userInfo } = useContext(UserContext);
+  const [loading, setLoading] = useState(true);
+
+  // Reload the component when the user info is fetched
+  useEffect(() => {
+    console.log("User info updated");
+    setLoading(false);
+  }, [userInfo]);
+
+  if (loading) {
+    return <div className="text-white text-center">Loading...</div>;
+  }
+
+  return (
+    <div className="w-gull h-screen bg-dark-gray">
+      <Navbar />
+      <Sidebar />
+      <h1 className="font-semibold text-5xl text-white text-center pt-32">
+        HomePage
+      </h1>
+      {/* <button className="self-center pt-2 px-4 py-2 bg-green text-white rounded items-center justify-center">
```

Comment:
Why is this commented out ?

Reply:
Was going to use it later on for reference. Should I delete?

Reply:
Actually it is pretty easy to recreate so it will be deleted.